src/diffusers/pipelines/shap_e/pipeline_shap_e.py

```diff
@@ -0,0 +1,358 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Should we not also include the OpenAI authors here and in the image to image one?